src/routes/docs/advanced/security/hippa/+page.markdoc

```diff
@@ -1,12 +1,28 @@
 ---
 layout: article
-title: HIPPA (Coming soon)
+title: HIPPA
```

Comment:
```suggestion
title: HIPAA
```

and in all other places

Reply:
Also the icon file name, to help avoid future confusion.

Reply:
Updated

Reply:
And updated elsewhere.